pkg/apis/core/validation/validation.go

```diff
@@ -4362,17 +4362,25 @@ func validatePodResources(spec *core.PodSpec, podClaimNames sets.Set[string], fl
 	if spec.Resources == nil {
 		return nil
 	}
+	resourcesFldPath := fldPath.Child("resources")
+	if spec.OS != nil && spec.OS.Name == core.Windows {
+		// Do not include more detailed errors on the resources field value
+		// if the resources field may not be set on the target OS.
+		return field.ErrorList{
+			field.Forbidden(resourcesFldPath, "may not be set for a windows pod"),
```

Comment:
Should "windows" be capitalized?

Reply:
I'd like to keep it as is for consistency with other existing validation messages, which also use the term "windows pods."